src/routes/usersRoute.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import UsersController from '../controllers/UsersController';
```

Comment:
You can just `import { signUp } from '......'`

Reply:
ok

Reply:
This is throwing up error, and I guess it is because it is a class based function and the methods are not exported, the class is exported default

See what I did `import { signUp } from '../controllers/UsersController';` 

 And I got this error

`Users/chidinmaezekwe/Desktop/test/merry-ah-backend/node_modules/express/lib/router/route.js:202
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]`
singUp bcomes undefined